src/GregClient/NuGet/tools/Install.ps1

```diff
@@ -0,0 +1,9 @@
+param($installPath, $toolsPath, $package, $project)
```

Comment:
This is a new powershell script that is added to the NuGet package to set the copy locals to false.